engine/src/test/scala/cromwell/engine/workflow/lifecycle/execution/ExecutionStoreBenchmark.scala

```diff
@@ -0,0 +1,52 @@
+package cromwell.engine.workflow.lifecycle.execution
```

Comment:
Is there CI support for running this and tracking changes over time? Maybe even giving us a red cross on the PR if we screw the benchmarks up too much (kinda like the code coverage stuff... that we ignore)